.github/CODEOWNERS

```diff
@@ -94,18 +70,10 @@
 
 /packages/teuchos/           @trilinos/teuchos
 
-#/packages/ThreadPool/        @trilinos/
-
 /packages/thyra/             @trilinos/thyra
 
 /packages/tpetra/            @trilinos/tpetra
 
-/packages/TriKota/           @trilinos/trikota
-
-#/packages/trilinoscouplings/ @trilinos/
```

Comment:
Same question for trilinoscouplings.

Reply:
Looks like there _is_ a trilinoscouplings team, I'll re-add this and assign them as a writer.